### PR TITLE
fix(ts-config): Add strict mode

### PR DESCRIPTION
**Pull request by leonardocartaxo**
fix(ts-config): strict mode error fix: Property <prop name> has no initializer and is not definitely assigned in the constructor.

### CODE REVIEW

File: packages/nestjs-email/src/email.service.ts [hunk 1]
```diff
@@ -19,6 +19,9 @@ export class EmailService {
     try {
       await this.mailerService.sendMail(dto);
     } catch (e) {
+      if (!(e instanceof Error)) {
```
Comment (MrMaz) [inline packages/nestjs-email/src/email.service.ts]: this should look like:

```
if (e instance of Error === false) {
  e = new NotAnErrorException(e);
}
// then if falls through to logger like normal
```

Comment (MrMaz) [inline packages/nestjs-email/src/email.service.ts]: this should look like:

```
if (e instance of Error === false) {
  e = new NotAnErrorException(e);
}
// then if falls through to logger like normal
```

File: packages/nestjs-logger/src/logger-transport.service.spec.ts [hunk 2]
```diff
@@ -138,7 +139,9 @@ describe('LoggerTransportService', () => {
   });
 
   it('loggerService.logLevel_null', () => {
```
Comment (MrMaz) [inline packages/nestjs-logger/src/logger-transport.service.spec.ts]: just remove that tests that are checking for null if strict mode already handles this

Comment (MrMaz) [inline packages/nestjs-logger/src/logger-transport.service.spec.ts]: just remove that tests that are checking for null if strict mode already handles this

File: packages/nestjs-email/src/email.service.ts (hunk 1 above)
Comment (leonardocartaxo) [reply packages/nestjs-email/src/email.service.ts]: we have this issue:
image

**Pull request merged**